packages/components/src/components/quickfact/quickfact.stories.ts

```diff
@@ -0,0 +1,249 @@
+import '../../solid-components';
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
+const { argTypes, parameters } = storybookDefaults('sd-quickfact');
+const { overrideArgs } = storybookHelpers('sd-quickfact');
+const { generateTemplate } = storybookTemplate('sd-quickfact');
+import '../../styles/display/display.css';
+import '../../styles/leadtext/leadtext.css';
+import '../../styles/paragraph/paragraph.css';
+import { html } from 'lit';
+import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport';
+import { userEvent } from '@storybook/test';
+import { waitUntil } from '@open-wc/testing-helpers';
+import { withActions } from '@storybook/addon-actions/decorator';
+
+export default {
+  title: 'Components/sd-quickfact',
+  component: 'sd-quickfact',
+  args: overrideArgs([
+    {
+      type: 'slot',
+      name: 'default',
+      value: `<div class="slot slot--border slot--text h-12">Default slot</div>`
+    },
+    {
+      type: 'slot',
+      name: 'summary',
+      value: `<div slot="summary"> <p class="text-base font-normal leading-normal  sm:text-3xl sm:leading-tight">Lorem Ipsum</p>
+            <div class="text-base font-normal leading-normal sm:text-xl">Con sectetur adipiscing elit</div></div>`
+    }
+  ]),
+
+  argTypes,
+  parameters: { ...parameters },
+  decorators: [withActions] as any
+};
+/**
+ * This shows sd-quickfact in its default state.
+ */
+
+export const Default = {
+  render: (args: any) => {
+    return generateTemplate({ args });
+  }
+};
+
+/**
+ * This shows sd-quickfact is displayed differently depending on whether or not the default slot is filled.
+ */
+
+export const OpenXContent = {
+  name: 'Open x Content',
+  render: (args: any) => {
+    return generateTemplate({
+      axis: {
+        y: {
+          type: 'slot',
+          name: 'default',
```

Comment:
```suggestion
          name: 'content',
```

Reply:
What is the suggestion about? We don't have a "content" slot, this is just the default slot which doesn't need further namings.

Reply:
its just about the naming of the axis. Not the slot itself. The current layout confuses me a bit as the axis is called `default`.